src/management/metadata.ts

```diff
@@ -0,0 +1,46 @@
+export interface ProjectMetadata {
```

Comment:
Perhaps FirebaseProjectMetadata to avoid confusion with Cloud projects and more closely match the proto name.

Reply:
Done